micro-rdk-nmea/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "micro-rdk-nmea"
+authors.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+rust-version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+micro-rdk = { path = "../micro-rdk", features = ["native"] }
```

Comment:
To my great annoyance, we've gone back to a state where micro-rdk cannot be built as a feature-less library. I'll make a ticket for this and link it here